scraper/core/server.py

```diff
@@ -89,12 +88,15 @@ def collect_data_from_sources():
         collect_data_from_sources starts the collection process by scraping data from various given sources.
         '''
         try:
+            LogMessage(f"Stepping into function.", LogMessage.LogTyp.INFO, SERVICENAME).log()
             data_list = [
-                    *Scraper.__get_data_from_rss_feed(),
+                    #*Scraper.__get_data_from_rss_feed(),
```

Comment:
Why do you disable the rss source?

Reply:
Just for testing. I will uncomment them.